frontend/src/components/signUpPage/InitialHeaderSignUp.jsx

```diff
@@ -0,0 +1,33 @@
+import { Link } from "react-router-dom"
```

Comment:
This component is exactly the same as InitialHeader.

Reply:
Done